### ISSUE TITLE
sudo allowances

### ISSUE THREAD

**Issue by brianjmurrell** [Wuerfelbecher/jellyfin-rpm#2]
I'm not sure that the existing sudo allowances should be allowed by default.  Perhaps they should be approved by the admin.

The restart permission I'm not so worried about but the `mount` permission seems a bit sketchy.

**Comment by Wuerfelbecher** [comment]
I agree with you on every point you made, since i mostly copied the [official debian package](https://github.com/jellyfin/jellyfin/tree/master/debian), which i even did wrong since i forgot to add the `-restartpath` option to the daemon, so the policy was unused.

I updated the .spec release to print to:

-  print a message after installation
-  moved the sudo policy to `/usr/share/jellyfin/jellyfin-sudoers` 
-  `/etc/sudoers.d/jellyfin` should be removed by RPM (due to spec bump) 
-  commented out the mount part of the policy

Thank you very much for pointing this out!

**Comment by brianjmurrell** [comment]
Looks good.  I really like the idea of using systemd's to detect when Jellyfin wants to restart itself.

Nice job getting this on RH distros by the way.  Really appreciate that.
